pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.554</version>
+    <version>1.580</version>
```

Comment:
Any reason why we should not go directly to plugin 2.X and use jenkins.version to control version to build against?